README.md

```diff
@@ -53,7 +53,7 @@ There are three ways of installing this library in your project.
     inputs.nixpkgs.url = "github:NixOS/nixpkgs";
 
     # add this line
-    inputs.sbt.url = "github:zaninime/sbt-derivation/refactor";
+    inputs.sbt.url = "github:zaninime/sbt-derivation/master";
```

Comment:
`master` being the default branch of the project, it is not necessary to specify it.

```suggestion
    inputs.sbt.url = "github:zaninime/sbt-derivation";
```